Aptoide/aptoide-client-v8
# [Feature/ASV-292] Home Fragment Metrics

**What does this PR do?**

   Home Fragment now has new metrics defined in [ASV-244](https://aptoide.atlassian.net/browse/ASV-244)

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

  Either debug FacebookEventLogger with breakpoints, use facebook developers dashboard or check log's in logcat regarding Analytics Manager and FacebookEventLogger.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-292](https://aptoide.atlassian.net/browse/ASV-292) [ASV-244](https://aptoide.atlassian.net/browse/ASV-244)

**Code Review Checklist**
